src/main/java/duke/commands/ListCommand.java

```diff
@@ -0,0 +1,23 @@
+package duke.commands;
```

Comment:
I like how you separated your files into multiple packages instead of one big package. Good job!

Reply:
Thanks!